### PR TITLE
Deploy Protocol Contracts on LightLink Mainnet and Testnet

### PR DESCRIPTION
**Pull request by vanshwassan**
Deployed and verified protocol contracts on [LightLink Mainnet](https://phoenix.lightlink.io/) and [LightLink Goerli Testnet](https://pegasus.lightlink.io/).

LightLink Phoenix Mainnet:

- [AccessControlRegistry](https://pegasus.lightlink.io/address/0x92E5125adF385d86beDb950793526106143b6Df1?tab=contract)

- [AirnodeRrpV0](https://pegasus.lightlink.io/address/0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd?tab=contract)

- [AirnodeRrpV0DryRun](https://pegasus.lightlink.io/address/0x2e768206bf5112e7D7efAf1d9df614C26475193f?tab=contract)

- [RequesterAuthorizerWithAirnode](https://pegasus.lightlink.io/address/0xf18c105D0375E80980e4EED829a4A68A539E6178?tab=contract)

LightLink Pegasus Testnet:

- [AccessControlRegistry](https://phoenix.lightlink.io/address/0x92E5125adF385d86beDb950793526106143b6Df1?tab=contract)

- [AirnodeRrpV0](https://phoenix.lightlink.io/address/0xa0AD79D995DdeeB18a14eAef56A549A04e3Aa1Bd?tab=contract)

- [AirnodeRrpV0DryRun](https://phoenix.lightlink.io/address/0x2e768206bf5112e7D7efAf1d9df614C26475193f?tab=contract)

- [RequesterAuthorizerWithAirnode](https://phoenix.lightlink.io/address/0xf18c105D0375E80980e4EED829a4A68A539E6178?tab=contract)

### CODE REVIEW

**Review by Ashar2shahid — APPROVED** [review]
Looks good to me, Lighlink recently reached out to integrate QRNG and we will have the providers redeploy to support the chain

**Review by dcroote — CHANGES_REQUESTED** [review]
Can you address the references.json merge conflict? The file / export changed recently in https://github.com/api3dao/airnode/pull/1929. It may only need rerunning of:
https://github.com/api3dao/airnode/blob/d3cf108d3ee49ef1c27762ba6ed4c6f90030dfaf/packages/airnode-protocol/package.json#L22C6-L22C6

I'm still confused on the path forward with  https://github.com/api3dao/chains/pull/137. Should that be reopened and merged, followed by a 4.3.0 release (4.2.0 is [already released](https://www.npmjs.com/package/@api3/chains?activeTab=versions))?

**Comment by Ashar2shahid** [comment]
> I'm still confused on the path forward with [api3dao/chains#137](https://github.com/api3dao/chains/pull/137). Should that be reopened and merged, followed by a 4.3.0 release (4.2.0 is [already released](https://www.npmjs.com/package/@api3/chains?activeTab=versions))?

Currently the chains CI runs on every chain in the package, there needs to be a flag to disable CI operation on a chain (for more fringe chains)

**Comment by Ashar2shahid** [comment]
I think RRP deployments should live in this chain for completeness sake, the dependency on @api3/chains should be figured out IMO rather than have RRP deployments live in some other repo. Alternatively we can have another repo "qrng-deployments" where we do all the rrp deployments if we do not want to support this.

**Comment by dcroote** [comment]
> Non-dAPI related chain additions to @api3/chains will create overhead for the dAPI operations

I thought api3/chains was the single source of truth for all chains and I lean this way:

> @api3/chains should be figured out IMO rather than have RRP deployments live in some other repo

Otherwise, we then have another source of chains and chain IDs to deal with. What could work is more streamlined or automated api3/chains release process to minimize the effort involved in adding a chain.

**Comment by andreogle** [comment]
I generally don't really follow what's going on with new chain integrations, but I'm happy to help streamline any processes with `@api3/chains` to make it easy to work with. I tried to make the repo as developer friendly as possible to encourage contributions/PRs. The guard rails (linting checks) are also fairly strict to prevent anything from going wrong.. Adding a new chain should be a fairly straightforward exercise at this point. I'm also happy to do same day releases assuming I'm available (otherwise someone else can help with this or we can automate it).

I assume [this comment](https://github.com/api3dao/chains/pull/137#issuecomment-1854520628) is the problematic one? I don't have much capacity at the moment to get too involved here, but it should be fairly straightforward to implement a new field that skips the chain in the CI. Happy to accept PRs if it's a blocker

**Comment by dcroote** [comment]
@andreogle - I find api3/chains quite dev friendly- didn't mean otherwise and apologies if it came off sounding that way. I created https://github.com/api3dao/chains/issues/153 and assigned myself to work on now so this can unblock folks.

**Comment by andreogle** [comment]
> didn't mean otherwise and apologies if it came off sounding that way

I didn't take it that way at all 😄 Thanks for implementing [the PR](https://github.com/api3dao/chains/pull/155) to skip provider checks 👍🏻 (cc @vanshwassan). 

I also just meant that I'm happy to try to help make the repo and/or process as streamlined as possible - whatever that might look like. I'm not a primary user of it so I don't always understand where the issues/blockers are.

**Comment by dcroote** [comment]
@vanshwassan - the [PR for skipping a chain in CI](https://github.com/api3dao/chains/pull/155) is merged so now https://github.com/api3dao/chains/pull/137 can be reopened, the new field added, merged, and api3/chains released. Then this PR, updated with the new api3/chains version, can be merged 😅

**Review by dcroote — APPROVED** [review]
I think the CI failures are because the PR is from your fork rather than from a branch within the repo and therefore lack the secrets necessary e.g. for pushing the docker containers. Next time can you use a branch? Otherwise LGTM

**Review by dcroote — CHANGES_REQUESTED** [review]
Actually I see the `yarn.lock` file hasn't updated- can you run `yarn run bootstrap` followed by `yarn run build`?

File: packages/airnode-protocol/package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "write-example-env-file": "hardhat run scripts/write-example-env-file.ts"
   },
   "devDependencies": {
-    "@api3/chains": "^4.1.1",
+    "@api3/chains": "^4.3.0",
```
Comment (dcroote) [inline packages/airnode-protocol/package.json:32]: Last thing (I'm pretty sure 😅) - can you make the same change to the `package.json` in `airnode-node` then rerun the yarn commands so there is a single version of `@api3/chains` in the monorepo?

Comment (vanshwassan) [reply packages/airnode-protocol/package.json:32]: Ah, probably missed that. Just updated it.

**Review by dcroote — APPROVED** [review]
Nice, and thanks for the patience on the small details

**Pull request merged**